opentracing/span.py

```diff
@@ -203,3 +205,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                                                 'val': exc_val,
                                                 'tb': exc_tb})
         self.finish()
+
+    def log_event(self, event, payload=None):
+        """DEPRECATED"""
+        return self
```

Comment:
even if deprecated, we can provide default behavior right here by delegating back to `log_kv`